src/components/DataTable/cells/TableCell.tsx

```diff
@@ -65,14 +30,17 @@ const TableCellForwardRef: React.ForwardRefRenderFunction<HTMLTableCellElement,
             className={clsx(
                 'moonstone-TableCell',
                 'textAlign' + capitalize(textAlign),
+                'flexRow',
+                'alignCenter',
                 'moonstone-verticalAlign' + capitalize(verticalAlign),
                 {flexFluid: typeof width === 'undefined'},
+                scrollableClass,
                 className
             )}
-            style={{width: width}}
+            style={{width, ...style}}
```

Comment:
I prefer to do it with `...props.style ` in order to avoid to declare another prop to just manage custom style